.cargo/audit.toml

```
@@ -10,4 +10,7 @@ ignore = [
   "RUSTSEC-2021-0139"  # ansi_term is no longer maintained, however this is a transient dependency of
```

Needs a comma to fix the syntax:

```suggestion
  "RUSTSEC-2021-0139",  # ansi_term is no longer maintained, however this is a transient dependency of
```